pkg/build/node/cni.go

```diff
@@ -172,6 +172,9 @@ metadata:
     app: kindnet
     k8s-app: kindnet
 spec:
+  selector:
```

Comment:
aside: this requirement to duplicate the labels exactly here is a really unfortunate aspect of the kubernetes API, should send feedback upstream about this